api/client/service/opts.go

```diff
@@ -199,14 +198,13 @@ func (m *MountOpt) Set(value string) error {
 			mount.Type = swarm.MountType(strings.ToLower(value))
 		case "source", "name", "src":
 			mount.Source = value
-		case "target", "dst", "dest", "destination", "path":
```

Comment:
https://www.google.ca/#q=dst

We don't need 4 aliases for the same thing. Adding something short to make for a more compact format is great. I think `dest` is short enough.